pkg/auth/rights/auth_info.go

```diff
@@ -47,3 +48,21 @@ func AuthInfo(ctx context.Context) (authInfo *ttnpb.AuthInfoResponse, err error)
 	}
 	return authInfo, nil
 }
+
+var errUnauthenticated = errors.DefineUnauthenticated("unauthenticated", "unauthenticated")
+
+// RequireAuthentication confirms if the authentication information within a context contains any rights, if so,
+// the user is considered to be authenticated.
```

Comment:
My only question is the term "user"; are you authenticating a user? Is this call always on behalf of a user? 
Or can it be made more generic by saying `call authentication` or `request authentication`.

Reply:
You are correct, It can be made a bit more generic.
Probably was a bit too focused on the interaction of importing the end devices via csv/json and ended up thinking the interaction only happens through a user.